SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -484,7 +499,11 @@ extension ViewController: TVOCallDelegate {
 extension ViewController: CXProviderDelegate {
     func providerDidReset(_ provider: CXProvider) {
         NSLog("providerDidReset:")
-        audioDevice.isEnabled = true
+        if let device = audioDevice as? TVODefaultAudioDevice {
+            device.isEnabled = true
+        } else if let device = audioDevice as? ExampleAVAudioEngineDevice {
+            device.isEnabled = true
+        }
```

Comment:
if and else are doing the same, can we remove the check and add comment saying `// ExampleAVAudioEngineDevice and DefaultAudioDevice both implements isEnabled `

Reply:
This one is tricky - both `TVODefaultAudioDevice` and `ExampleAVAudioEngineDevice` implement `enabled` but not the `TVOAudioDevice` interface, which is the type of the `audioDevice` iVar. The `as` type-casting seems to be the only way here so that I can access `isEnabled` of the device, unless we have separate iVars declared as explicit types.
@timrozum need your suggestion here.